### ISSUE TITLE
Add alt attributes to images

### ISSUE THREAD

**Issue by kosarko** [ufal/dspace-angular#10]
Non-text content should have text-alternative ([WCAG 1.1.1](https://www.w3.org/TR/WCAG21/#non-text-content))

There are images with no alt tag.

A (maybe too coarse as it ignores `[attr.alt]`) search across component templates
```
find ./src/ -type f -name '*.html' -exec bash -c "xmllint --html --xpath '//img[not(@alt)]' {} >/dev/null 2>&1" \; -print
```
reports these files:

[src/static-files/deposit.html](../blob/7899a88/src/static-files/deposit.html)
[src/static-files/cs/deposit.html](../blob/7899a88/src/static-files/cs/deposit.html)
[src/app/header/header.component.html](../blob/7899a88/src/app/header/header.component.html)
[src/app/admin/admin-sidebar/admin-sidebar.component.html](../blob/7899a88/src/app/admin/admin-sidebar/admin-sidebar.component.html)
[src/app/thumbnail/thumbnail.component.html](../blob/7899a88/src/app/thumbnail/thumbnail.component.html)
[src/app/home-page/home-page.component.html](../blob/7899a88/src/app/home-page/home-page.component.html)
[src/app/shared/clarin-item-box-view/clarin-item-box-view.component.html](../blob/7899a88/src/app/shared/clarin-item-box-view/clarin-item-box-view.component.html)
[src/app/shared/comcol/comcol-page-logo/comcol-page-logo.component.html](../blob/7899a88/src/app/shared/comcol/comcol-page-logo/comcol-page-logo.component.html)
[src/app/logout-page/logout-page.component.html](../blob/7899a88/src/app/logout-page/logout-page.component.html)
[src/themes/custom/app/login-page/login-page.component.html](../blob/7899a88/src/themes/custom/app/login-page/login-page.component.html)
[src/themes/custom/app/logout-page/logout-page.component.html](../blob/7899a88/src/themes/custom/app/logout-page/logout-page.component.html)
[src/themes/dspace/app/navbar/navbar.component.html](../blob/7899a88/src/themes/dspace/app/navbar/navbar.component.html)

some of those might be false positives (see the `[attr.alt]` note above), maybe there's a better way to search for these.
